Enable inductor CI for huggingface

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #87423
* __->__ #86792

Summary: Unit tests will be enabled after fixed in trunck. TorchBench and TIMM need
more setup and are coming later.

cc @jansel @lezcano @fdrocha @mlazos @soumith @voznesenskym @yanboliang

desertfire committed:
Enable inductor CI

Summary: This enables running inductor's tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

jansel commented:
@desertfire you can also add an auto-labeler rule to turn on these tests if people touch dynamo/inductor:
https://github.com/pytorch/pytorch/blob/master/.github/labeler.yml

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire commented:
> @desertfire you can also add an auto-labeler rule to turn on these tests if people touch dynamo/inductor: https://github.com/pytorch/pytorch/blob/master/.github/labeler.yml

This one is actually `TEST_CONFIG` based, and the plan is to have inductor unit tests running for all PRs. I will use the auto-labeler for integral tests later.

desertfire commented:
> > @desertfire you can also add an auto-labeler rule to turn on these tests if people touch dynamo/inductor: https://github.com/pytorch/pytorch/blob/master/.github/labeler.yml
> 
> This one is actually `TEST_CONFIG` based, and the plan is to have inductor unit tests running for all PRs. I will use the auto-labeler for integral tests later.

With https://github.com/pytorch/pytorch/pull/87228, inductor unit tests do run for all PRs. I will update this PR to include auto-labeler and add more tests.

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor's unit tests. Enable inductor on
pytorch tests are still ongoing.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor unit tests and model tests under
the ciflow/inductor label.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor unit tests and model tests under
the ciflow/inductor label.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor unit tests and model tests under
the ciflow/inductor label.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor unit tests and model tests under
the ciflow/inductor label.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor unit tests and model tests under
the ciflow/inductor label.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI"


Summary: This enables running inductor unit tests and model tests under
the ciflow/inductor label.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI on huggingface and timm models"


Summary: Unit tests will enabled after fixed in trunck. TorchBench test needs
more setup and is coming in another PR.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI on huggingface and timm models"


Summary: Unit tests will enabled after fixed in trunck. TorchBench test needs
more setup and is coming in another PR.

[ghstack-poisoned]

desertfire committed:
Update on "Enable inductor CI on huggingface and timm models"


Summary: Unit tests will enabled after fixed in trunck. TorchBench test needs
more setup and is coming in another PR.

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -h

desertfire commented:
@pytorchbot -h merge

huydhn reviewed (approved):
LGTM!

jansel commented on .jenkins/pytorch/common_utils.sh:
```diff
@@ -170,6 +170,18 @@ function test_torch_deploy(){
+function install_timm() {
+  pip_install pandas
+  pip_install scipy
+  pip_install git+https://github.com/rwightman/pytorch-image-models
```
Suggest pinning to a specific version

desertfire committed:
Update on "Enable inductor CI for huggingface"


Summary: Unit tests will be enabled after fixed in trunck. TorchBench and TIMM need
more setup and are coming later.

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -g

desertfire committed:
Update on "Enable inductor CI for huggingface"


Summary: Unit tests will be enabled after fixed in trunck. TorchBench and TIMM need
more setup and are coming later.

[ghstack-poisoned]

desertfire commented:
[pytorchbot](https://github.com/pytorchbot) merge -g

desertfire commented:
@pytorchbot merge -g
